oneview/resource_server_profile_template.go

```diff
@@ -141,6 +155,7 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 		SerialNumberType:   d.Get("serial_number_type").(string),
 		WWNType:            d.Get("wwn_type").(string),
 		MACType:            d.Get("mac_type").(string),
+		Description:        d.Get("description").(string),
```

Comment:
I think these fields are not mandatory fields. In that case these attributes have to read using GetOk. Can you please recheck on this in documentation?